Hi3Helper.Core/Classes/Shared/Region/LauncherConfig.cs

```diff
@@ -23,45 +24,163 @@ public struct CDNURLProperty : IEquatable<CDNURLProperty>
+
+        private static void InitScreenResSettings()
+        {
+            ScreenProp.InitScreenResolution();
+            GetScreenResolutionString();
+        }
+        #endregion
+
+        #region CDN List
+        public static List<CDNURLProperty> CDNList => new List<CDNURLProperty>
+                                                      {
+                                                          new CDNURLProperty
+                                                          {
+                                                              Name = "GitHub",
+                                                              URLPrefix = "https://github.com/neon-nyan/CollapseLauncher-ReleaseRepo/raw/main",
+                                                              Description = Lang._Misc.CDNDescription_Github,
+                                                              PartialDownloadSupport = true
+                                                          },
+                                                          new CDNURLProperty
+                                                          {
+                                                              Name = "Cloudflare",
+                                                              URLPrefix = "https://r2.bagelnl.my.id/cl-cdn",
+                                                              Description = Lang._Misc.CDNDescription_Cloudflare,
+                                                              PartialDownloadSupport = true
+                                                          },
+                                                          new CDNURLProperty
+                                                          {
+                                                              Name = "Bitbucket",
+                                                              URLPrefix = "https://bitbucket.org/neon-nyan/collapselauncher-releaserepo/raw/main",
+                                                              Description = Lang._Misc.CDNDescription_Bitbucket
+                                                          }
+                                                      };
 
         public static CDNURLProperty GetCurrentCDN() => CDNList[GetAppConfigValue("CurrentCDN").ToInt()];
+        #endregion
 
+        #region Misc Fields
         public static Vector3 Shadow16 = new Vector3(0, 0, 16);
         public static Vector3 Shadow32 = new Vector3(0, 0, 32);
         public static Vector3 Shadow48 = new Vector3(0, 0, 48);
         // Format in milliseconds
         public static int RefreshTime = 250;
 
-        const string SectionName = "app";
-        public static string startupBackgroundPath;
+        const         string       SectionName = "app";
+        public static string       startupBackgroundPath;
         public static List<string> ScreenResolutionsList = new List<string>();
 
+        public const long AppDiscordApplicationID     = 1138126643592970251;
```

Comment:
Are we sure that we just want to expose application IDs just like that? It allows someone to spoof their application using our ID, which is a potential security risk.

Reply:
There isn't a known Discord ApplicationID abuse cases known since its quite useless as it only allows for an RPC showing in your profile and doesn't interact with anything more than that. In fact, some project like PreMID has all their possible ApplicationID exposed as well so I don't believe this is an issue.